### PR TITLE
Creating an offboarding

### PR DESCRIPTION
**Pull request by estherrgarcia**
Work in progress with Anna

### CODE REVIEW

File: talent/offboarding.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 This document describes the off-boarding process at source{d}.
 We believe that the off-boarding process should be smooth and clear in order to cause minimal disruption to every including member.
 
-The relationship with the company can finish in three different ways. 
+The relationship with the company can finish in two different ways. 
 
 #### The employee decides to leave
```
Comment (tsolakoua) [inline talent/offboarding.md]: The employee leaves voluntarily

File: talent/offboarding.md [hunk 2]
```diff
@@ -29,3 +29,49 @@ When every document is signed, the last day, the office manager has to:
  
  
  
+#### The company decides to fire the employee
```
Comment (tsolakoua) [inline talent/offboarding.md]: The employee leaves involuntarily

File: talent/offboarding.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
-The relationship with the company can finish in three different ways. 
+The relationship with the company can finish in two different ways. 
 
 #### The employee decides to leave
 
-- The employee has to tell the company (usually) 15 days in advance. 
-- The employee has to sign a resignation letter with their information and the end date. Example. 
+- The employee has to inform the company (usually) 15 days in advance. 
+- The employee has to sign a resignation letter with their information and the end date. [Example](https://drive.google.com/file/d/0B6AGEF9RyVDxdEFULWx1dzNSd0E/view?usp=sharing).
```
Comment (tsolakoua) [inline talent/offboarding.md]: The employee has to sign a resignation letter with their information and the end date, as shown in the [example](https://drive.google.com/file/d/0B6AGEF9RyVDxdEFULWx1dzNSd0E/view?usp=sharing))

File: talent/offboarding.md [hunk 4]
```diff
@@ -1,16 +1,16 @@
...
 
-- The employee has to tell the company (usually) 15 days in advance. 
-- The employee has to sign a resignation letter with their information and the end date. Example. 
+- The employee has to inform the company (usually) 15 days in advance. 
+- The employee has to sign a resignation letter with their information and the end date. [Example](https://drive.google.com/file/d/0B6AGEF9RyVDxdEFULWx1dzNSd0E/view?usp=sharing). 
 - The company has to give the accountant the following information to prepare the settlement: 
   - End date.
   - Type of leaving: voluntary severance.
-  - Holidays to enjoy:the office manager has to check in Kin HR. 
+  - Holidays to enjoy: the office manager has to check in Kin HR.
```
Comment (tsolakoua) [inline talent/offboarding.md]: Holidays to enjoy -> Vacations left 
(maybe sounds more formal like that)

File: talent/offboarding.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+We believe that the off-boarding process should be smooth and clear in order to cause minimal disruption for each person.
+
+The relationship with the company can finish in two different ways: 
+
+#### The employee leaves voluntarily
+
+_This process describes employees working for the Spanish entity, for the USA this is not yet documented._
+
+- The employee has to inform the company, usually 15 days in advance. 
+- The employee has to sign a resignation letter with their information and the end date, as shown in the [example] (https://drive.google.com/file/d/0B6AGEF9RyVDxdEFULWx1dzNSd0E/view?usp=sharing).
```
Comment (dpordomingo) [inline talent/offboarding.md]: I could not find a resignation letter in that link.

Comment (tsolakoua) [reply talent/offboarding.md]: It's this one as far as I know: Papeleta de conciliación individual @dpordomingo

**Pull request merged**